pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -733,25 +752,33 @@ public synchronized String toString(LocalTime localTime) {
 
     sbuf.setLength(0);
 
-    if (localTime.equals( LocalTime.MAX )) {
+    if (localTime.isAfter(MAX_TIME)) {
       return "24:00:00";
     }
 
+    int nano = localTime.getNano();
+    if (nano % 1000 >= 500) {
```

Comment:
This expression (`nano % 1000 >= 500`) is used in more than one place, why don't extract it to a function?

Reply:
Any ideas for the function name?

Reply:
`isNanoCarry`?
`isNanoOverflow` may be misleading.